src/screens/splashScreen/SplashScreen.tsx

```diff
@@ -11,8 +11,8 @@ import {
 } from 'react-native';
 import SplashScreen from 'react-native-splash-screen';
 
-import myImage from '../../assets/launchScreen.png';
-import COLORS from '../colors/myColors.tsx';
+import myImage from '../../../assets/launchScreen.png';
```

Comment:
@muhammadhassan2k3 can you please use meaningfull name for it instead of `myImage` ?

it could be called as `launchScreen`

Reply:
@muhammadhassan2k3 it is still myImage. Rename it to something meaningful.